### ISSUE TITLE
Add `--host` param to phonegap remote

### ISSUE THREAD

**Issue by wildabeast** [phonegap/phonegap-cli#579]
This is mostly selfish at the moment so that we at PhoneGap Build can push to a test server -- however if alternatives to PGB pop up one day (if we were to say open source Build), then this would become useful. I'll try to get some time to do this myself if y'all can't get to it, but tracking here for now.

**Comment by mwbrooks** [comment]
I think this is totally reasonable. Most of the underlining architecture is abstracted to support other end-points, so I'll look into adding this.

@wildabeast any particular deadline?

**Comment by mwbrooks** [comment]
Published `phonegap@5.5.0` with this feature.

I wasn't able to test it on a separate end-point, so let me know if you run into any issues.

All of the remote commands now have `--protocol`, `--host`, `--port`, `--path`, and `--proxy` exposed.

For example:

```
$ phonegap remote login --help

Usage: phonegap remote login [options]

Description:

  Log into your PhoneGap/Build account.

Options:

  -u, --username       your username (email address)
  -p, --password       your password
  --protocol           server protocol (default: 'https:')
  --host               server host (default: 'build.phonegap.com')
  --port               server port (default: '443')
  --path               server path prefix (default: '/api/v1')
  --proxy              proxy server to route requests
```

`phonegap@5.5.0` made the following changes https://github.com/phonegap/phonegap-cli/compare/5.4.1+cordova.5.4.1...5.5.0+cordova.5.4.1

`phonegap-build@0.10.0` made the following changes https://github.com/phonegap/node-phonegap-build/compare/0.9.2...0.10.0

`phonegap-build-api@0.4.0` made the following changes https://github.com/phonegap/node-phonegap-build-api/compare/0.3.3...0.4.0
